src/patternfly/components/Toolbar/toolbar.scss

```diff
@@ -0,0 +1,153 @@
+
+// The value of component scoped variables is always defined by a global variable:
+.pf-c-toolbar {
+  --pf-c-toolbar--PaddingTop: var(--pf-global--spacer--md);
+  --pf-c-toolbar--PaddingRight: var(--pf-global--spacer--xl);
+  --pf-c-toolbar--PaddingBottom: var(--pf-global--spacer--md);
+  --pf-c-toolbar--PaddingLeft: var(--pf-global--spacer--xl);
+  --pf-c-toolbar--child--MarginLeft: var(--pf-global--spacer--md);
+
+  // Bulk Select
+  --pf-c-toolbar__bulk-select--MarginRight: var(--pf-global--spacer--md);
+
+  // Filter
+  --pf-c-toolbar__filter--MarginTop: calc(var(--pf-c-toolbar--PaddingBottom) + var(--pf-global--spacer--md));
+  --pf-c-toolbar__filter--MarginLeft: 0;
+
+  // Filter toggle toggle
+  --pf-c-toolbar__filter-toggle--MarginLeft: var(--pf-global--spacer--md);
+  --pf-c-toolbar__filter-toggle--m-expanded--c-button--Color: var(--pf-global--Color--100);
+
+  // Sort
+  --pf-c-toolbar__sort--MarginLeft: var(--pf-global--spacer--sm);
+
+  // Action group
+  --pf-c-toolbar__action-group--child--MarginLeft: var(--pf-global--spacer--md);
+  --pf-c-toolbar__sort--action-group--MarginLeft: var(--pf-global--spacer--sm);
+  --pf-c-toolbar__filter--action-group--MarginLeft: var(--pf-global--spacer--xl);
+
+  // Action list
+  --pf-c-toolbar__action-list--MarginLeft: var(--pf-global--spacer--sm);
+  --pf-c-toolbar__sort--action-list--MarginLeft: var(--pf-global--spacer--sm);
+
+  // Filter list
+  --pf-c-toolbar__filter-list--MarginTop: var(--pf-global--spacer--md);
+  --pf-c-toolbar__filter-list--c-button--MarginLeft: var(--pf-global--spacer--sm);
+
+  flex-wrap: wrap;
+  padding: var(--pf-c-toolbar--PaddingTop) var(--pf-c-toolbar--PaddingRight) var(--pf-c-toolbar--PaddingBottom) var(--pf-c-toolbar--PaddingLeft);
+
+  // default spacing between children
+  > * + * {
+    margin-left: var(--pf-c-toolbar--child--MarginLeft);
+  }
+}
+
+.pf-c-toolbar,
+.pf-c-toolbar__filter-list,
+.pf-c-toolbar__action-group {
+  display: flex;
+  align-items: center;
+}
+
+// Bulk Select
+.pf-c-toolbar__bulk-select {
+  margin-right: var(--pf-c-toolbar__bulk-select--MarginRight);
+}
+
+// Filter input group
+.pf-c-toolbar__filter-toggle {
+  --pf-c-toolbar--child--MarginLeft: var(--pf-c-toolbar__filter-toggle--MarginLeft);
+
+  &.pf-m-expanded {
+    color: var(--pf-c-toolbar__filter-toggle--m-expanded--Color);
+  }
+
+  .pf-c-button {
+    display: flex;
+    align-items: center;
+  }
+
+  @media screen and (min-width: $pf-global--breakpoint--lg) {
+    display: none;
+  }
+}
+
+.pf-c-toolbar__filter {
+  flex-basis: 100%;
+  order: 1;
+  width: auto;
+  margin-top: var(--pf-c-toolbar__filter--MarginTop);
+  margin-left: var(--pf-c-toolbar__filter--MarginLeft);
+
+  @media screen and (max-width: $pf-global--breakpoint--lg) {
+    display: none;
+
+    &.pf-m-expanded {
+      display: flex;
+    }
+  }
+
+  @media screen and (min-width: $pf-global--breakpoint--lg) {
+    --pf-c-toolbar__filter--MarginTop: 0;
+    --pf-c-toolbar__filter--MarginBottom: 0;
```

Comment:
I don't see this being used.

Reply:
yep, margin between this and filter (chip) list is on the filter list now.